inc/class_parser.php

```diff
@@ -368,8 +368,8 @@ function cache_mycode()
 
 		if($mybb->settings['allowfontmycode'] == 1)
 		{
-			$nestable_mycode['font']['regex'] = "#\[font=([a-z0-9 ,\-_'\"]+)\](.*?)\[/font\]#si";
-			$nestable_mycode['font']['replacement'] = "<span style=\"font-family: $1;\" class=\"mycode_font\">$2</span>";
+			$nestable_mycode['font']['regex'] = "#\[font=([\\\"']?)([a-z0-9 ,\-_]+)\\1\](.*?)\[\/font\]#si";
```

Comment:
Doesn't look like escaping the forward slash is needed in `[\/font\]`

Reply:
True. Will remove shortly.